applications/GeoMechanicsApplication/custom_processes/apply_k0_procedure_process.cpp

```diff
@@ -22,16 +22,46 @@
 #include "utilities/math_utils.h"
 
 // Application includes
+#include "custom_constitutive/linear_elastic_law.h"
 #include "geo_mechanics_application_variables.h"
 #include "includes/kratos_flags.h"
-#include "includes/kratos_parameters.h"
+
+namespace
+{
+
+using namespace Kratos;
+
+void SetConsiderDiagonalEntriesOnlyAndNoShear(ModelPart::ElementsContainerType& rElements, bool Whether)
```

Comment:
I think the AndNoShear referred to the last diagonal entries. Those are now left intact, thus making this a less descriptive name.

Reply:
Mistake of Wijtze Pieter, checked again in the code and the G term on the (XY, XY) diagonal really becomes 0. Ignore the remark.